gprofiler/profilers/perf.py

```diff
@@ -178,32 +179,41 @@ def __init__(
         self._node_processes_attached: List[Process] = []
         self._perf_memory_restart = perf_memory_restart
         switch_timeout_s = duration * 3  # allow gprofiler to be delayed up to 3 intervals before timing out.
+        extra_args = []
+        try:
+            extra_args.extend(
```

Comment:
Let's log the chosen event (default / cpu-clock / cpu-cycles) here.

Reply:
we could change `get_perf_event_args` so that it will return `SUPPORTED_PERF_EVENTS` instead of `List`, and then log the event itself instead of the picked list.
I think that somewhat nicer, lmk what you think

Reply:
f3cecb035fbcc92b524744f9539a81a3fd64022e